lib/features/family/features/profiles/screens/profile_selection_screen.dart

```diff
@@ -165,17 +166,17 @@ class _ProfileSelectionScreenState extends State<ProfileSelectionScreen> {
                             const SizedBox(height: 8),
                             if (state.profiles.length >= 3) ...[
                               FunButton(
-                                isTertiary: true,
                                 onTap: () => context.goNamed(
```

Comment:
why no? 😢

Reply:
It's now FunButton.tertiary() and FunButton.secondary() :)